theseus/geometry/point_types.py

```diff
@@ -62,6 +85,29 @@ def __init__(
         dof, data = _prepare_dof_and_data(3, data)
         super().__init__(dof=dof, data=data, name=name, dtype=dtype)
 
+    @staticmethod
+    def rand(
+        *size: int,
+        generator: Optional[torch.Generator] = None,
+        dtype: Optional[torch.dtype] = None,
+        device: Optional[torch.device] = None,
+        requires_grad: Boolean = False,
+    ) -> "Point3":
+        if len(size) != 1:
+            raise ValueError("The size should be 1D.")
+        return Point3(
+            data=2
+            * torch.rand(
```

Comment:
Why limit to [-1, 1]?

Reply:
No reasons. I can change it to any range.

Reply:
See comment in the discussion of `rand` and `randn`.

Reply:
Change the range of `rand()` for `Vector`, `Point2`, `Point3` and `SE2` to `[0, 1]`